static/app/views/performance/http/httpSamplesPanel.tsx

```diff
@@ -94,11 +108,17 @@ export function HTTPSamplesPanel() {
     transaction: query.transaction,
   };
 
+  if (query.responseCodeClass) {
+    filters['span.status_code'] = `${query.responseCodeClass}*`;
```

Comment:
l: it might be faster to have this be served from the backend and avoid using the wildcard search with `*` (`span.status_code_class`?)